### PR TITLE
Add legacy backend x86 testing.

### PR DESCRIPTION
**Pull request by jashook**
Uses the x86_legacy_backend exclude file.

### CODE REVIEW

**Comment by jashook** [comment]
@mmitche PTAL

**Comment by mmitche** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -213,7 +213,11 @@ def static addPRTrigger(def job, def architecture, def os, def configuration, is
                                     {
                                         buildCommands += "build.cmd ${lowerConfiguration} ${architecture}"
                                     }
-                                    // TEMPORARY. Don't run tests for PR jobs on x86
+                                    // x86 has a different config file.
+                                    if (architecture == 'x86' || !isPR) {
```
Comment (mmitche) [inline netci.groovy]: This looks like it would run tests twice for x64.  Is that what you want?

**Comment by jashook** [comment]
That was not the desired behavior, I believe that change should address that.

File: netci.groovy [hunk 2]
```diff
@@ -231,13 +234,13 @@ def static addPRTrigger(def job, def architecture, def os, def configuration, is
                                     // For windows, pull full test results and test drops for x86/x64
                                     Utilities.addArchival(newJob, "bin/Product/**,bin/tests/tests.zip")
                                     // TEMPORARY. Don't run tests for PR jobs on x86
-                                    if (architecture == 'x86' && !isPR) {
-                                        Utilities.addXUnitDotNETResults(newJob, 'bin/**/TestRun*.xml')
-                                    }
+                                    //if (architecture == 'x86' && !isPR) {
```
Comment (mmitche) [inline netci.groovy]: Just remove these lines

**Comment by jashook** [comment]
Closing for now, looks like regressions have been introduced into legacy backend that either need to addressed or excluded before this change is merged.

**Pull request closed without merging**